book-domain-service/config_test.py

```diff
@@ -0,0 +1,14 @@
+import os 
+
+class Config_Test(object):
+
+    BASEDIR = os.path.abspath(os.path.dirname(__file__))
```

Comment:
You often see a `BASEDIR` constant like this defined at the module level rather that in the config object itself unless you wanted to have it be a configurable option based on the environment.

Reply:
In this case the configurable option is for the SQLALCHEMY_DATABASE_URI. To create path for the test database. Maybe there is a better constant name then BASEDIR. I attempted to extract the BASEDIR to the __init__ file and it created a circular dependency. For now I can rename it ?

Reply:
I think the file it is in is the correct place, I just typically see it outside of the config object as a file level constant. Something like this: https://github.com/miguelgrinberg/flasky/blob/master/config.py#L2

(Although I think it should be all caps because it is a constant)